admin/data/sql/migrations/083.do.create-pupil-census-table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE [mtc_admin].[pupilCensus] (
```

Comment:
as part of this script, you will need to execute the stored procedure that binds the audit trigger because you are introducing a new table

Reply:
the stored procedure is called `mtc_admin.spGenAuditTriggers`

Reply:
also, this table should have a unique index on blobFileName

Reply:
@GuyHarwood aside from the audit trigger execution everything else has been changed as requested

Reply:
@GuyHarwood additionally I included some new fields relevant to validation, completion (bit) and error message as a prep for the next pbi. 
Let me know your thoughts

Reply:
This looks like it *ought* to be a general purpose async job system, rather than something tightly coupled to pupil census.

It should allow for a wide range of async jobs, and provide a queue system so a job worker knows which job has priority.